tasks-servers-ioc.yml

```diff
@@ -1,9 +1,12 @@
 ---
+- name: Include distribution-dependent variables
+  include_vars: 
+    file: "group_vars/{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml"
```

Comment:
For more robustness we could add something like this:

```
- name: Include distribution-dependent variables
  include_vars: '{{ item }}'
  with_first_found:
    - files:
        - '{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml'
        - '{{ ansible_distribution }}.yml'
        - '{{ ansible_os_family }}.yml'
      ignore_errors: true
      paths:
        - 'group_vars'
```

So we can have fallbacks for distributions